wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -111,6 +122,35 @@ h4{
 
 }
 
+.ggrc-btn-white-lg{
+    background: #FFFFFF;
+    border: 5px solid #0B4F6D;
+    box-sizing: border-box;
+    border-radius: 50px;
+    padding: 26.5px 67px;
+    font-size: 22px;
+    color: #0B4F6D;
+    ;
```

Comment:
?

Reply:
I remember you once mentioned to have css definitions for different types of button as per the figma styling [here](https://www.figma.com/file/vEG8pWuftMli0PwYMeQj2j/GGRC-design?node-id=3293%3A19848) . So there are 3 sizes and 2 colors for background so that's how I tried naming - ggrc-btn-[background color]-[size] . Sizing I'm doing lg for large, md for medium and sm for small. I tried working on it now as I'm coming across the button from discussion boards onwards and will eventually change others. Is that okay or you want me to do it any other way?